vsintegration/src/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -300,10 +300,10 @@ type internal FsiToolWindow() as this =
                 strHandle.Free()
         )
 
-    let executeTextNoHistory sourceFile (text:string) =
+    let executeTextNoHistory sourceFile (text: unit -> string) =
         sessions.Ensure(sourceFile)
         textStream.DirectWriteLine()
-        sessions.SendInput(text)
+        sessions.SendInput(text())
```

Comment:
```suggestion
        sessions.SendInput(text)
```

Reply:
I'm afraid it is a function call. we need to delay getting the value for SupportsInteractivePrompt until after we have started fsi.exe, otherwise the string may be composed before fsi has been started.